src/main/java/nl/tudelft/jpacman/game/AnimationLevelObserver.java

```diff
@@ -0,0 +1,28 @@
+package nl.tudelft.jpacman.game;
+
+import nl.tudelft.jpacman.level.Level;
+import nl.tudelft.jpacman.sprite.WinAnimation;
+
+public class AnimationLevelObserver implements Level.LevelObserver {
+    private Game game;
+    private WinAnimation animation;
+
+    public AnimationLevelObserver(Game game, WinAnimation animation) {
+        this.game = game;
+        this.animation = animation;
+    }
+
+    @Override
+    public void levelWon(Level level) {
+        WinJudge judge = new WinJudge(2000,0);
+
+        if(judge.isEpicWin(game.getStepsTaken(), level.getTotalPellets())) {
+            animation.playAnimation();
+        }
+    }
+
+    @Override
+    public void levelLost(Level level) {
```

Comment:
`Observers` is just so meh in Java, Guava's `EventBus` would be a perfect fit. The amount of reflection is however a bit too high for the purpose of this framework I am afraid.